### ISSUE TITLE
Applications that pipe the screen output like nano and less don't display anything

### ISSUE THREAD

**Issue by ghost** [zeit/hyper#2957]
- [X] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [X] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10 Build 1709
- **Hyper.app version**: Hyper 2.1.0-canary.1
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: N/A
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla

## Issue
Basically when I run something like `less .zshrc` or `nano .zshrc` the screen turns black, yet it accepts inputs and I can (using nano for example) type something, save and exit and it returns properly to the normal prompt.

This issue happens with both bash and zsh running the WSL. Trying this in the Windows cmd.exe works perfect so it is an issue of Hyper itself.

**Comment by hummal** [comment]
Have an similar problem with hyper 2.0.0

If I use debug outputs of a nodejs application it wont diplay anything. Instead the output is random. Every time I repeat it differs. Have a look at the screenshot.
![image](https://user-images.githubusercontent.com/633711/39572474-90462178-4ecf-11e8-9644-a141fdc3838c.png)

If I use the native bash.exe in Windows it displays the full output every time.

**Comment by JBlond** [comment]
I use 2.0.0 on Windows 10. After closing vim the content of vim sill remains on the screen. Same goes for nano.

![vim](https://user-images.githubusercontent.com/2391810/40165486-6741ddba-59bc-11e8-9308-1f095b7ee756.png)

Using clear on the console move the cursor up to the top, but the previous content is sill visible, it only gets overwritten line by line that I type new.

![clear](https://user-images.githubusercontent.com/2391810/40165589-ab9ddd7e-59bc-11e8-9036-3fa8f9227061.png)
